README.md

```diff
@@ -76,14 +46,15 @@
 2. [BDR 01](./decisions/bdr-01.md)
 
 ## Team Information
+
 <!--- Template Instructions  
   Provide a brief description of your team, how it came to be, etc.
 --->  
 
 Our team members came together because ...
```

Comment:
```suggestion
Our team members came together because we share a passion for accessibility, design systems, and the modernization of tech stacks. We believe that these combined passions give our team a competitive advantage and the strength to make hackathon contributions with a real impact!
```